server/src/db/models/users.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+  email: {

```

Comment:
Are we just using email, or also storing a token of some sorts?

Reply:
I think email would be enough? @wshin387 is just email enough? like when then authenticate and the middleware says ok, can I just then take the email? or should we store the token somewhere too